src/components/Board/BoardShare/BoardShare.component.js

```diff
@@ -26,6 +27,12 @@ import messages from './BoardShare.messages';
 
 import './BoardShare.css';
 
+function OpenLogWd(){
+  window.open(window.location.origin + '/login-signup','',"width=500,height=800");
```

Comment:
You have to use react router to move to a different location inside the application, see an example here:
https://github.com/cboard-org/cboard/blob/51035b0a503a9f7c3903e13fcd05f2895e89f57c/src/components/Board/Navbar/Navbar.js#L172

React router: https://reactrouter.com/web/guides/quick-start